### DISCUSSION TITLE
AWS resource tags

### DISCUSSION THREAD

**Discussion by faaik** [architect/architect#1391] [General]
Is it possible to add tags to the AWS resources using `arc` configuration or another way?

**Comment by ryanblock** [comment]
It is! Please see `arc deploy --tags ...` ([more here](https://arc.codes/docs/en/reference/cli/deploy)); you can also fully customize tags with [`deploy.start` plugins](https://arc.codes/docs/en/guides/plugins/deploy#deploy.start).

See also: #31

**Reply by faaik** [reply]
Thank you 🙏

**Comment by nathanielpomianowski** [comment]
What is the best way to determine if this works correctly?

In my case, I am not seeing the tags showing up in the Resource Groups & Tag Editor.

Examples of what I'm doing. I believe I am implementing this correctly. Is there something I'm missing?

gitlab-ci.yml.
```
npm run deploy -- --staging --prune --tags Name=Phantom
```

package.json script
```
"deploy": "arc deploy"
```

**Reply by nathanielpomianowski** [reply]
I found a solution.

If you have multiple tags they must be wrapped by quotes.

i.e.
```
npm run deploy -- --staging --prune --tags "Name=Phantom Boo='Moo hoo' Foo=1235r2314"
```
